cfpb/capital-framework
# Add cf-notification

Fixes https://github.com/cfpb/capital-framework/issues/440

## Additions

- Adds cf-notification 1.0.

## Testing

- Pull this down, run `npm run build`
- `cd tmp/cf-notification`
- `npm link`
- Change to another CF repo and `git checkout gh-pages-notifications`.
- Run `npm link cf-notification`
- Run `npm run build && npm start`
- Check http://localhost:3000/components/cf-notification/

## Screenshots

<img width="937" alt="screen shot 2017-08-14 at 8 57 43 pm" src="https://user-images.githubusercontent.com/704760/29298092-57e65786-8133-11e7-87be-78f98a9fb40f.png">
<img width="940" alt="screen shot 2017-08-14 at 8 57 52 pm" src="https://user-images.githubusercontent.com/704760/29298093-57f55b46-8133-11e7-815c-323a9fe31116.png">
<img width="915" alt="screen shot 2017-08-14 at 8 57 57 pm" src="https://user-images.githubusercontent.com/704760/29298094-57f62742-8133-11e7-8ff0-4733d9dbc2f2.png">
